docs/ios/native-dependencies.md

```diff
@@ -0,0 +1,39 @@
+# Native dependencies
+
+## Add a new external native dependency
+
+All iOS code is kept in `ios` folder and using the same `Podfile` that's why process of adding a new external native dependency can be achieved by running a command:
```

Comment:
This is a bit unclear to me. I would simply title this paragraph: "Adding a React Native dependency", because `link` will not work with pure CocoaPods modules not downloaded from `npm`. This may get confusing.